src/main/java/org/osiam/ResourceServer.java

```diff
@@ -67,13 +55,16 @@
     @Value("${org.osiam.resource-server.db.vendor}")
     private String databaseVendor;
 
-    @Bean
-    public static PropertySourcesPlaceholderConfigurer properties() {
-        PropertySourcesPlaceholderConfigurer propertySources = new PropertySourcesPlaceholderConfigurer();
-        Resource[] resources = new ClassPathResource[] { new ClassPathResource("resource-server.properties") };
-        propertySources.setLocations(resources);
-        propertySources.setIgnoreUnresolvablePlaceholders(true);
-        return propertySources;
+    public static void main(String[] args) {
+        System.setProperty("spring.jpa.hibernate.naming_strategy", "org.hibernate.cfg.ImprovedNamingStrategy");
+        SpringApplication.run(ResourceServer.class, args);
+    }
+
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
+        return application
+                .showBanner(true)
+                .sources(ResourceServer.class);
```

Comment:
The docs state for this method:

> Add more sources (configuration classes and components) to this application

So is this really necessary? At least this class is indirectly annotated with `@Configuration` via `@SpringBootApplication`.